### ISSUE TITLE
запуск приложения

### ISSUE THREAD

**Issue by IvanKhristenko** [DEGTEVUWU/DocumentAccounting2#1]
не удалось запустить проект указанными в Readme командами :
```
sudo docker-compose up --build -d
или
make run
```
у меня почему то заработал только такой вариант : 
![Screenshot from 2024-07-23 14-25-42](https://github.com/user-attachments/assets/8e80998b-4cfb-4b71-936d-33629598a8f3)
 
тоесть получается дефис в команде лишний

**Comment by DEGTEVUWU** [comment]
да, это странно, так как у меня работают оба вариант, но вот при деплое отрабатывает только вариант с дефисом. Скажи, пж, только в этом была проблема с запуском? В итоге все поднялось и работало?

**Comment by IvanKhristenko** [comment]
только в этом. без дефиса все работате
